### PR TITLE
Kyle K's suggestions

### PR DESCRIPTION
**Pull request by milkie**
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/2794)

### CODE REVIEW

File: source/reference/read-concern.txt [hunk 1]
```diff
@@ -52,9 +52,9 @@ The following read concern levels are available:
    * - .. readconcern:: "linearizable"
 
      - The query returns data that reflects all successful writes
-       issued with a :writeconcern:`"majority"` *and* acknowledged
+       issued with a write concern of :writeconcern:`"majority"` *and* acknowledged
        prior to the start of the read operation. Linearizable read
-       concern returns data that cannot be rolled back. You can specify
+       concern returns data that will never be rolled back. You can specify
```
Comment (milkie) [inline source/reference/read-concern.txt]: Where do we say w:majority write concern writes could roll back? I'll check everything for continuity.

Comment (kay-kim) [inline source/reference/read-concern.txt]: https://docs.mongodb.com/manual/reference/write-concern/#acknowledgement-behavior
https://docs.mongodb.com/manual/reference/replica-configuration/#rsconf.writeConcernMajorityJournalDefault
https://docs.mongodb.com/manual/core/replica-set-rollbacks/#avoid-replica-set-rollbacks

Comment (milkie) [inline source/reference/read-concern.txt]: Oh right.  The Note applies to this blurb about linearizable reads as well.

Comment (milkie) [inline source/reference/read-concern.txt]: Perfect.

**Comment by kay-kim** [comment]
merged!

**Pull request closed without merging**